initrd/bin/mount-usb

```diff
@@ -55,9 +55,11 @@ if [ -z ${USB_MOUNT_DEVICE} ]; then
     # remove block device from list if numeric partitions exist, since not bootable
     let USB_NUM_PARTITIONS=`ls -1 $i* | wc -l`-1
     if [ ${USB_NUM_PARTITIONS} -eq 0 ]; then
-      echo $i >> /tmp/usb_disk_list
+      echo $i $(blkid | grep $i | grep -o 'LABEL=".*"' | sed 's/\ UUID.*$//') >> /tmp/usb_disk_list
```

Comment:
alternate formatting:  
echo "$i ($(blkid | grep $i | grep -o 'LABEL=".*"' | cut -f2 -d '"'))" >> /tmp/usb_disk_list

could even swap the order/positioning to put the label first

Reply:
@MrChromebox : yes!!!! Lable first would be awesome